scripts/backup.sh

```
@@ -0,0 +1,61 @@
+    mc mirror local/$bucket $BACKUP_DIR/$MINIO_BACKUP_DIR/$bucket
+done
+
+# Comprimir backups
+echo "🗜️  Comprimindo backups..."
+cd $BACKUP_DIR
+tar -czf "apae_backup_$DATE.tar.gz" $POSTGRES_BACKUP_FILE $MINIO_BACKUP_DIR
+cd ..
+
+# Limpar arquivos temporários
+rm -f $BACKUP_DIR/$POSTGRES_BACKUP_FILE
+rm -rf $BACKUP_DIR/$MINIO_BACKUP_DIR
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)